### ISSUE TITLE
Differentiate repeating parameter(AND) from single parameter with multiple values(OR)?

### ISSUE THREAD

**Issue by navndn** [Asymmetrik/node-fhir-server-core#172]
### Do you want to request a feature, report a bug, or improve documentation?
A feature request, probably, not sure if this is a bug.

Currently, both  `/Patient?language=FR&language=NL` (logical `AND` ) and  `/Patient?language=FR,NL` (logical `OR` ) produce same content in `args`
```json
{
    "base_version": "4_0_0",
    "language": "FR,NL"
}
```
The `AND` or `OR` info is lost. 

Shouldn't the `AND` scenario content be as follows?
```json
{
    "base_version": "4_0_0",
    "language":  [ "FR", "NL" ]
}
```

**What OS are you using and what version of node.js and @asymmetrik/node-fhir-server-core are you running?**

Docker base image: node:10-stretch
Node.js: v10.15.3
node-fhir-server-core: 1.4.0

Thanks!

**Comment by Robert-W** [comment]
Hi @navndn,

Yes this is one of the current limitations. We developed a tool we intended to release soon, its not in 2.0.0 because we didn't want to introduce too many breaking changes at once. We mentioned it in our [2.0.0 Migration Guide](https://github.com/Asymmetrik/node-fhir-server-core/blob/master/docs/MIGRATION_2.0.0.md#sanitize-params) as something currently missing but definitely on the roadmap. 

However, in 2.0.0, it passes the original request into your services, and express does expose them in that manner. So if you access the original req.query, you would see that `language=FR&language=NL` exposes `{language: ['FR','NL']}` and `language=FR,NL` exposes `{language: 'FR,NL' }`
